GitUI/Theming/ThemePersistence.cs

```diff
@@ -57,89 +53,124 @@ public void Save(Theme theme, string fileName)
             static int ToRbgInt(Color с) => с.ToArgb() & 0x00ffffff;
         }
 
-        private bool TryParse(
+        private bool TryReadFile(string fileName, out string result)
+        {
+            result = null;
+            var fileInfo = new FileInfo(fileName);
+
+            if (!fileInfo.Exists)
+            {
+                PrintTraceWarning(fileName, _fileNotFound.Text);
+                return false;
+            }
+
+            // > 1MB
+            if (fileInfo.Length > (1 << 20))
```

Comment:
nit While I have no problem with this myself, regular C# programmers may have a problem understanding...

Reply:
true :)